packages/@expo/cli/src/api/rest/wrapFetchWithProxy.ts

```diff
@@ -1,4 +1,4 @@
-import createHttpsProxyAgent from 'https-proxy-agent';
```

Comment:
Drop this from the package.json and update the CLI docs to note that we don't use this library anymore (as of SDK 52).

Reply:
Done!